src/components/Results/utils.ts

```diff
@@ -44,10 +52,11 @@ const getKeywords = (study: ResearchStudy): string[] => study.keyword?.map(({ te
 
 const getLikelihood = (search: BundleEntrySearch): LikelihoodProps => {
   const score = search?.score;
-  if (score >= 0.75) return { text: 'High-likelihood match', color: 'common.green' };
-  else if (score >= 0.01) return { text: 'Possible match', color: 'common.yellow' };
-  else if (score < 0.01) return { text: 'No match', color: 'common.red' };
-  else return { text: 'Unknown likelihood', color: 'common.grayLight' };
+  if (score >= 0.75) return { text: 'High-likelihood match', color: 'common.green', score };
+  else if (score >= 0.01) return { text: 'Possible match', color: 'common.yellow', score };
+  else if (score < 0.01) return { text: 'No match', color: 'common.red', score };
+  // Even if a trial doesn't have a likelihood, it might be better for the patient to decide whether it's a match than for them to never see it
+  else return { text: 'High-likelihood match', color: 'common.green', score: 1 };
```

Comment:
I don't know if I agree with a ResearchStudy defaulting to score 1 and High-likelihood match (IE, putting it at the top). I can be swayed either way given our current matching services, but should the absence of a score place it at the very top? 

@zlister Please comment.

Reply:
Joyce and I had this conversation. In theory, all of our current matching services that are used should have a score. This should be handled by the wrapper before the results gets to the app. So I guess if there is a worst case that a trial slips through the cracks, it should be a possible match. I don't think it should be at the end just in case there are no matches that are returned. 

Ultimately, let's make it a 0.5 instead of a score of 1. Does that make sense?

Reply:
I'll adjust the default score to 0.5.